forms-flow-api-utils/src/formsflow_api_utils/utils/permissions.json

```diff
@@ -0,0 +1,70 @@
+[{

```

Comment:
We can keep this in a python file as a Dict and avoid file operation. File operations are expensive in resource usage.